onnx/__init__.py

```diff
@@ -210,15 +212,15 @@ def _deserialize(s: bytes, proto: _Proto) -> _Proto:
 
 
 def load_model(
-    f: Union[IO[bytes], str],
+    f: Union[IO[bytes], str, os.PathLike],
     format: Optional[Any] = None,  # pylint: disable=redefined-builtin
     load_external_data: bool = True,
 ) -> ModelProto:
     """Loads a serialized ModelProto into memory.
 
     Args:
         f: can be a file-like object (has "read" function) or a string containing a file name
-        format: for future use
+        format or a pathlike object: for future use
```

Comment:
```suggestion
        format: for future use
```

Misplaced?

Reply:
I may be misunderstanding the English utilized here, but doesn't "for future use" here refer to some sort of usage related thing. I am attempting to imply here that a pathlike object is also a valid input.